lib/storage/file.js

```diff
@@ -1019,12 +1019,18 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.responseDisposition - If you provide this value, 
+ *     the response-content-disposition parameter of the signed url is set 
+ *     accordingly.
+ * @param {string=} options.responseType - If you provide this value, the 
+ *     response-content-type parameter of the signed url is set accordingly.
  * @param {function=} callback - The callback function.
  *
  * @example
  * file.getSignedUrl({
  *   action: 'read',
- *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14) // 2 weeks.
+ *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14), // 2 weeks.
+ *   responseDisposition: 'attachment; filename="filename.ext"'
```

Comment:
Providing both sounds like the best choice.
I would avoid the exception as it is not a severe error. Can we let `responseDisposition` win and maybe log a warning?